view/frontend/templates/instant/hit.phtml

```diff
@@ -26,7 +26,7 @@ $tierFormatedVar = 'price' . $priceKey . '_tier_formated'
                             <div class="result-sub-content">
                                 <div class="product-reviews-summary short">
                                     <div class="rating-summary">
-                                        <span class="label"><span>Rating:</span></span>
+                                        <span class="label"><span><?php echo __('Rating') ?>:</span></span>
```

Comment:
I'm checking the `en_US.csv` file and "Rating" is not translated there. Can you add it? 🙏

Reply:
I added Rating and As low as! Thanks for the catch!